pkgs/development/python-modules/black/default.nix

```diff
@@ -23,14 +23,19 @@ buildPythonPackage rec {
 
   # Don't know why these tests fails
   # Disable test_expression_diff, because it fails on darwin
```

Comment:
This might be a good time to switch to ``pytestCheckHook``. See https://github.com/NixOS/nixpkgs/blob/6770ab0771703b573a0a1c1f6c238352bebc9c31/doc/languages-frameworks/python.section.md#using-pytestcheckhook

This would allow flexible disabling of tests for darwin-only. Also setting LC_ALL or PATH could happen at ``preCheck``